src/components/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -180,20 +173,19 @@ const calendarRemountKey = ref<number>(0)
 const defaultTimeRange: TimeRange = {
   start: null,
   end: null,
-  timePeriodsKey: '',
 }
 
 /**
  * Dynamically choose the v-model
  * Single date is a Date, whereas a Date range is an object containing `start` and `end` dates
  */
-const calendarSingleDate = ref<Date | null>(modelValue.start)
+const calendarSingleDate = ref<Date | null>(modelValue.start || null)
 const calendarRange = ref<TimeRange>(modelValue || defaultTimeRange)
 const calendarVModel = isSingleDatepicker.value
   ? calendarSingleDate as DatePickerModel
   : calendarRange as DatePickerModel
 
-const calendarWrapperRef = useTemplateRef('calendarWrapperRef')
+const calendarWrapper = useTemplateRef('calendarWrapperRef')
```

Comment:
Rename this variable. Was seeing readonly warning when useTemplateRef has same variable name as template ref. https://github.com/vuejs/core/commit/bc63df01992fdbf0b6749ad234153725697ed896 

Happens in older vue versions